packages/react-scripts/template/src/Components/Landing.js

```diff
@@ -4,7 +4,10 @@ export default class Landing extends React.Component {
   render() {
     return (
       <div className="landing">
-        <p>‍👨‍🚀</p>
+        <section className="easter-egg">
```

Comment:
Misuse of <section> el.
"The <section> tag defines sections in a document, such as chapters, headers, footers, or any other sections of the document."